mautic/mautic
# TPROD 312/TPROD 351 - CoreBundle Helper templates cleanup

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR is to remove the unused CoreBundle Helper php templates, and replace references to these Helper files in other places. There is also some minor overlap with [this cleanup PR](https://github.com/mautic/mautic/pull/12032/)
Some of these changes include:
- removal of some MauticSocialBundle templates that were previously reverted, including index file (haven't seen issues)
- removal of some Standard templates



#### Steps to test this PR:
- Since the Helper files are used just about everywhere, probably all pages need to be checked.
- some of the helpers are responsible for pagination, modals, lists, tables, buttons, tabs, …


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.
